iquip/apps/scheduler.py

```diff
@@ -166,10 +176,13 @@ class ExperimentModel(QAbstractListModel):
         experimentData: The list of ExperimentView widget.
     """
 
-    def __init__(self, data: List[ExperimentInfo], parent: Optional[QWidget] = None):
-        """Extended."""
+    def __init__(self, parent: Optional[QWidget] = None):
+        """Extended.
+        
+        Args:
+            parent: The widget that contains this model."""
         super().__init__(parent)
-        self.experimentData = data
+        self.experimentData = []
```

Comment:
Since the initialization does not give any information of the element type, so you can annotate it like:
```python
self.experimentData: List[ExperimentView] = []
```
By the way, why is its name "...Data", not "...Views"?